party3.tex

```diff
@@ -0,0 +1,35 @@
+\section{Incorporating third party code } \label{sec:party3}
+
+The dev guide has a procedure \footnote{\url{https://developer.lsst.io/stack/packaging-third-party-eups-dependencies.html}}
+for making a third part package for inclusion in the pipelines distrib. In the future they may be other distributions as well.
+This procedure should be slightly modified such that the RFC is always flagged for CCB approval.
+
+A distribution is defined in \secref{sec:distribution} an example distribution would be the docker container available as the kernel in the notebook aspect of the science platform. A collaboration may wish to get some  non LSST code included in that distribution.
+
+There are several reasons to include third part code in our distributions:
+\begin{enumerate}
+\item There are packages which we use in our code e.g. starlink\_ast.  These are standard dependencies.  Currently these are put in lsst github org and probably should not be. Apart from that this is a well understood reason for third party code. \label{item:depend}
+\item There are packages which we want to use when working with the deployed software. Pandas  might be such an example - we don't need it to build or run our code but its sure handy for interaction with the data.\label{item:want}
+\item A collaborator has a package which depends on our code and is intended for some form of post processing. Another example here would be an algorithm we want to use such as ngmix which we do not necessarily need to support. This is what most people on LSST are thinking about under the topic of third party software. \label{item:colab}
```

Comment:
I don't agree with "This is what most people on LSST are thinking about under the topic of third-party software." This sentence doesn't really add anything either. Can we talk about this?

Reply:
This section is probably going to be moved in a different document.